ui/selectors/selectors.js

```diff
@@ -350,22 +364,21 @@ export function getInternalAccount(state, accountId) {
   return state.metamask.internalAccounts.accounts[accountId];
 }
 
-export function getInternalAccountsSortedByKeyring(state) {
-  const accounts = getInternalAccounts(state);
+export const getInternalAccountsSortedByKeyring = createSelector(
```

Comment:
Can you add the documentation?

Reply:
Will address this in a small follow up pr